canvas_modules/common-canvas/locales/common-properties/locales/es.json

```diff
@@ -66,7 +66,9 @@
   "expression.recently.used": "Usados recientemente",
   "expression.all.functions": "Todas las funciones",
   "expression.min.label": "Mín",
+  "expression.minimize.label": "Minimize",
```

Comment:
You don't need to update these files.  It will default back to en automatically.